amy/dashboard/tests/test_get_involved_views.py

```diff
@@ -0,0 +1,234 @@
+from datetime import date
+
+from django.urls import reverse
+
+from trainings.models import Involvement
+from workshops.models import TrainingProgress, TrainingRequirement
+from workshops.tests.base import TestBase
+
+
+class TestGetInvolvedCreateView(TestBase):
+    def setUp(self):
+        super()._setUpUsersAndLogin()
```

Comment:
Please be aware that this is using an admin user to perform the requests. Perhaps it would be better to create an instructor account and authorize it in `self.client`.